src/skmultiflow/meta/online_under_over_bagging.py

```diff
@@ -147,6 +145,9 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
         passed in the first partial_fit call, or if they are passed in further
         calls but differ from the initial classes list passed..
         """
+        if self.adwin_ensemble is None:
```

Comment:
Should not `self.ensemble` be used here, instead? (to cope with other ensemble models)

Reply:
Correct, fixed in 0d7aaeeb2841b9e5dbfe80fb821691ee2fbbd166